src/base/bittorrent/peerinfo.cpp

```diff
@@ -408,3 +408,8 @@ QString PeerInfo::flagsDescription() const
 {
     return m_flagsDescription;
 }
+
+int PeerInfo::downloadingPieceIndex() const
```

Comment:
No, it is not enough because I use `TorrentInfo::filesForPiece()` when I generate a tooltip for the torrent progress bar. There are no peers at this point.